test/e2e/healthcheck/healthcheck_panic_test.go

```diff
@@ -0,0 +1,100 @@
+package healthcheck_test
+
+import (
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/kumahq/kuma/pkg/config/core"
+	. "github.com/kumahq/kuma/test/framework"
+)
+
+var _ = Describe("Test HealthCheck panic threshold", func() {
+
+	var universalCluster Cluster
+	var deployOptsFuncs []DeployOptionsFunc
+
+	healthCheck := `
+type: HealthCheck
+name: hc-1
+mesh: default
+sources:
+- match:
+    kuma.io/service: '*'
+destinations:
+- match:
+    kuma.io/service: echo-server_kuma-test_svc_8080
+conf:
+  interval: 10s
+  timeout: 2s
+  unhealthyThreshold: 3
+  healthyThreshold: 1
+  healthyPanicThreshold: 61
+  failTrafficOnPanic: true
+  tcp: {}`
+
+	dp := func(idx int) string {
+		return fmt.Sprintf(`
+type: Dataplane
+mesh: default
+name: dp-echo-%d
+networking:
+  address: 192.168.0.%d
+  inbound:
+  - port: 8080
+    servicePort: 80
+    tags:
+      kuma.io/service: echo-server_kuma-test_svc_8080
+      kuma.io/protocol: http`, idx, idx)
+	}
+
+	BeforeEach(func() {
+		universalCluster = NewUniversalCluster(NewTestingT(), Kuma1, Verbose)
```

Comment:
Should it be `Verbose`, when in other tests we are using `Silent`?

Reply:
Yep, I did it for debugging, changed to `Silent` thanks!